.ci/containers/github-differ/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM alpine/git
+from golang:1.18-stretch as resource
```

Comment:
just noting the change: this moves us from alpine -> debian as a base image, substantially increasing image size. https://hub.docker.com/_/golang notes that the golang alpine images are experimental and not officially supported, however.

Reply:
I would be fine with using the experimental golang:1.18-alpine images as long as they're compatible with our dependencies. I.e. if trying to use them results in compilation errors I don't think it's worth our time to try to get past that unless we can show that there would be a substantial expected speedup in our pipelines. But if it works as a drop-in replacement it seems like an easy win.

Reply:
Everything works as is. We use this image for other containers.